blueflood-http/src/main/java/com/rackspacecloud/blueflood/http/DefaultHandler.java

```diff
@@ -16,14 +16,33 @@
 
 package com.rackspacecloud.blueflood.http;
 
+import com.codahale.metrics.Timer;
+import com.rackspacecloud.blueflood.io.Constants;
+import com.rackspacecloud.blueflood.utils.Metrics;
+import org.jboss.netty.buffer.ChannelBuffers;
 import org.jboss.netty.channel.ChannelHandlerContext;
-import org.jboss.netty.handler.codec.http.HttpRequest;
-import org.jboss.netty.handler.codec.http.HttpResponseStatus;
+import org.jboss.netty.handler.codec.http.*;
 
 public class DefaultHandler implements HttpRequestHandler {
+    private static final Timer sendResponseTimer = Metrics.timer(DefaultHandler.class, "HTTP Ingestion response sending timer");
```

Comment:
@tilogaat ok thats what I thought. can you fix the name of the timer to make it not ingestion specific.